doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -24,33 +24,99 @@ mvn install
++
+. Complete one of the following actions to ensure that the {KIE_SERVER} loads the necessary KJAR modules:
+** To configure {KIE_SERVER} to scans and deploy all KJAR modules available in the Spring Boot application, add the following property to the `application.properties` file:
++
+[source]
+----
 kieserver.autoScanDeployments=true
 ----
 +
+When this property is set to `true`, {KIE_SERVER} deploys all KJAR modules available in the application, whether they are declared programmatically or through the Maven plug-in.
++
+This option is the simplest method to include all KJAR modules. However, it has two drawbacks:
++
+--
+*** The application sets all container IDs and aliases automatically, based on the group, artifact, and version (GAV) of every KJAR module. You cannot set a custom container ID or alias for a KJAR module.
+*** At startup time, the application scans the JAR file and the class path for KJAR modules. Therefore, the duration of startup might be increased.
+--
++
+To avoid these drawbacks, you can configure every KJAR module individually using the `application.properties` file or using Java source code, as described in one of the following options.
++
+** To configure every KJAR module individually using the `application.properties` file, for each of the KJAR modules that you want to include in the service, add the following properties to the `application.properties` file:
++
+[source]
+----
+kieserver.deployments[<n>].containerId=<container>
+kieserver.deployments[<n>].alias=<alias>
+kieserver.deployments[<n>].artifactId=<artifact>
+kieserver.deployments[<n>].groupId=<group>
+kieserver.deployments[<n>].version=<version>
+----
++
+Replace the following values:
++
+--
+*** `<n>`: A sequential number: `0` for the first KJAR module, `1` for the second module, and so on
+*** `<container>`: The container ID for the KJAR module
+*** `<alias>`: The alias for the KJAR module
+*** `<artifact>`: The artifact ID for the KJAR module
+*** `<group>`: The group ID for the KJAR module
+*** `<version>`: The version ID for the KJAR module
+--
++
+The following example configures two versions of the `Evaluation` KJAR module:
++
+[source]
+----
+kieserver.deployments[0].alias=evaluation_v1
+kieserver.deployments[0].containerId=evaluation_v1
+kieserver.deployments[0].artifactId=Evaluation
+kieserver.deployments[0].groupId=com.myspace
+kieserver.deployments[0].version=1.0.0-SNAPSHOT
 
-[cols="30%,15%,55%", options="header"]
-|===
-|Parameter
-|Values
-|Description
+kieserver.deployments[1].alias=evaluation_v2
+kieserver.deployments[1].containerId=evaluation_v2
+kieserver.deployments[1].artifactId=Evaluation
+kieserver.deployments[1].groupId=com.myspace
+kieserver.deployments[1].version=2.0.0-SNAPSHOT
+----
++
+** To configure every KJAR module individually using Java source code, create the `KieContainerDeployer` class in your business application service, similar to the following example:
```

Comment:
`KieContainerDeployer` class is just a java class example name. The important thing here are the methods declared inside the class, thus this class name can be any other name. So maybe I would re-phrase that sentence to something like:

```suggestion
** To configure every KJAR module individually using Java source code, create a class in your business application service, similar to the following example:
```